ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -109,6 +109,12 @@ let prototyped_of_message = function
       Some "22.20.0"
   | "Repository", "set_gpgkey_path" ->
       Some "22.12.0"
+  | "PCI", "get_dom0_access_status" ->
+      Some "24.13.0-next"
```

Comment:
If we commit these now then will they be correctly updated the next time we tag it?